src/_scss/pages/interactiveDataSources/_scrollerOverlay.scss

```diff
@@ -0,0 +1,36 @@
+.scroller-overlay {
+  // width: 100%;
+  min-height: 100vh;
+  display: flex;
+  align-items: center;
+  padding: 0px 30px;
+  h3 {
```

Comment:
Weird to do this for only one header specifically. If we need this, guessing we need it for all of them? Or maybe we can just drop it

Reply:
I think we should leave it in case we want to put headers inside the scroller overlay card and h3 seems to be the right font size without any additional styles.